packages/dd-trace/src/platform/node/proto.js

```diff
@@ -30,11 +30,18 @@ class Client {
 
     this._prefix = options.prefix || ''
     this._tags = options.tags || []
-    this._clientToken = options.clientToken || ''
-    this._componentName = options.componentName || ''
+    this._accessToken = this._accessTokenFromTags(this._tags)
+    this._componentName = options.service || ''
```

Comment:
if the service is the same, I would remove the `this._componentName` but then pls update `proto.js` and make sure the `service` will be use instead (to create report)